types/akamai-edgeworkers/test/cookies-tests.ts

```diff
@@ -0,0 +1,51 @@
+import { Cookies, SetCookie } from 'akamai-edgeworkers/cookies';
```

Comment:
This doesn't seem to match how these would be referenced in the akamai edgeworkers samples: https://github.com/akamai/edgeworkers-examples/blob/f273d82ca7186e08345ab65d383db4e723a03282/cookie-accrual/main.js#L11

Reply:
I believe I ran into problems with the build when I included a path mapping in the tsconfig.json. I'll try switching it back to the regular convention and see what Travis has to say.

Reply:
Now that the extra modules are defined in the index.d.ts, they can be referenced without the extra class. Please ignore my previous comment.